akka-projection-core/src/main/scala/akka/projection/TelemetrySpi.scala

```diff
@@ -95,7 +102,9 @@ trait Telemetry {
 
   override def stopped(): Unit = {}
 
-  override def afterProcess(readyTimestampNanos: Long): Unit = {}
+  override def beforeProcess(): AnyRef = Unit
```

Comment:
Invoked when creating the `ProjectionContext` instance so the time measures implemented in Cinnamon would be identical to the ones we had before this commit.